contributors/contributors/community-edi.md

```diff
@@ -0,0 +1,23 @@
+# Community Equity, Diversity, and Inclusion (EDI)
+ 
+We're committed to purposefully designing EDI into every facet of our programs and operations. This section is meant to document tips and best practices that we strive to follow as we plan events and activities.
+
+## Event organization best practices
+
+1. **Tools:** Think purposefully about the tools we are using to engage our audience. For example, some people in the open source world may not be able to use tools that are proprietary software. Also, some people in certain countries may not be able to use certain tools because of country/region restrictions. Sending a survey beforehand with different tool options could help identify good tools to use for a specific event.
+2. **Timezones:** Be as inclusive of as many different timezones as possible.
+3. **Code of Conduct:** Make our Code of Conduct a requirement at all events and actively help our community with friction and conflict.
+4. **Accessibility:** Advocate for accessibility in all of our events and materials.
+5. **Captioning and translation:** Strive to have all video captioned in at least American English, and translated in as many languages as possible. For help looking for translators, [***INSERT STUFF HERE***]
+6. **Share slides beforehand:** Share slides with attendees so that people who use screen readers can follow along.
+7. **Images should have alt text:** This is for screen readers for people who are blind or are visually impaired. Be descriptive with the image. Maybe a few sentences about what the most important content is.
+8. **Use proper headings in your editor:** Use your editor's headings so that screen readers understand the organization of the page.
```

Comment:
```suggestion
8. **Use proper headings in your text editor:** Use your text editor's built-in heading formatting so that screen readers understand the organization of the page.
```